### PR TITLE
Popup modal for Desktop and Mobile view

### PR DESCRIPTION
**Pull request by OyePriscilla**
- Following interactions implemented:
  - When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
  - When the user clicks (or taps) the close (X) button, the popup disappears.
  - You should implement popups for both mobile and desktop screen sizes:
  - Mobile popup

- In order to associate each project with the popup window details, JavaScript array was used to store all of the information for the projects.
  - For each project we store the following pieces of data in a JavaScript object, at a minimum: name, description, featured image, technologies, link to live version, link to source.
  -We updated the main page so that the projects section is created dynamically using the information stored in that JavaScript object.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -5,6 +5,37 @@ const close = document.querySelector('.close-icon');
...
+const mobileModalButtonThree = document.querySelector('.btn-touch-three');
+const mobileModalButtonFour = document.querySelector('.btn-touch-four');
+const closeMobileButton = document.querySelector('.close-mobile-button');
+
+mobileModalButtonOne.addEventListener('click', () => {
+  document.querySelector('.desktop-mode').style.display = 'none';
+  document.querySelector('.mobile-modal-container').style.display = 'block';
+});
+
+mobileModalButtonTwo.addEventListener('click', () => {
```
Comment (koachcode) [inline main.js:20]: - hi @OyePriscilla Good job implementing the project thus far, Please on the desktop view the dynamic display of projects does not work as it is empty, please try fixing it. 

##### Your design 

https://user-images.githubusercontent.com/58771507/174150151-3b1cb869-2308-4f87-bfed-89f441e7afac.mp4

File: main.js [hunk 2]
```diff
@@ -25,3 +56,93 @@ overlayLinkTwo.addEventListener('click', () => {
...
+// Portfolio projects in an array
+const projects = [
+  {
+    id: 1,
+    name: 'Tonic',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    featured_img: './images/desktop-works/work-one.png',
+    tecnologies: ['html', 'css', 'javascript'],
+    live_link: 'https://github.com/OyePriscilla/MyPortfolioSetup',
```
Comment (koachcode) [inline main.js:69]: - Please on your mobile view the popup functionality does not work, please check it out and fix it.

If you find this feedback confusing, please do reach out to me on slack `@kanu mike chibundu`

**Review by koachcode — CHANGES_REQUESTED** [review]
Hi @OyePriscilla , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
you-are-near

## Highlights
- No linter errors :+1: 
- Good commit message :+1: 
- Good Branch name :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OLIPLICHE — APPROVED** [review]
Hi @OyePriscilla , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**